<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathView</class>
 <widget class="QMainWindow" name="MathView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Aloevera</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MmadelaiCalc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">s21--Button, #arrow,#special,#x_value {
border-image: url(:/SeaMelody/Key.png);
}

s21--ColorButton {
border-image: url(:/SeaMelody/ColorKey.png);
}

#degree , #mul {
text-align:top;
padding-top:25px;
}

QPushButton:pressed,QToolButton:pressed{
color:rgb(37,150,190);
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="equals">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>71</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>80</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#equals {
	background-color: transparent;
	border-image: url(:/SeaMelody/VerticalKey.png);
	padding-bottom:95;
	padding-left:5;
}

#equals:pressed{
color:rgb(37,150,190);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>25</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#zero {
border-image: url(:/SeaMelody/Zero.png);
text-align:left, top;
padding-left:15;
padding-top:12;
}
#zero:pressed{
color:rgb(37,150,190);
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="s21::Button" name="one">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="s21::Button" name="two">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="s21::Button" name="three">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="s21::Button" name="four">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="s21::Button" name="five">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="s21::Button" name="six">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="s21::Button" name="seven">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="s21::Button" name="eight">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="s21::Button" name="nine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QToolButton" name="special">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_area">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>30</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#text_area {
background-color: transparent;
border-image: url(:/SeaMelody/text_area.png);
background: none;
border: none;
background-repeat: none;
color:rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>        Made by Mmadelai &lt;3</string>
    </property>
    <property name="maxLength">
     <number>255</number>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="plus">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>60</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="minus">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>60</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="mul">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>64</pointsize>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="div">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>40</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="mod">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="s21::Button" name="l_bracket">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="s21::Button" name="r_bracket">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="s21::Button" name="sqrt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="degree">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>64</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>^</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="arrow">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="x_sym">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>410</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="ln">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="log">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="sin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="cos">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="acos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="tan">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>18</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="s21::ColorButton" name="atan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QToolButton" name="graph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#graph {
border-image: url(:/SeaMelody/SpecialKey.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="s21::Button" name="dot">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aloevera</family>
      <pointsize>80</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_value">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>-10000000000000000470601344959054695891559601407866630764278709534898249531392.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000000000043845843045076197354634047651840.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <zorder>nine</zorder>
   <zorder>equals</zorder>
   <zorder>zero</zorder>
   <zorder>one</zorder>
   <zorder>two</zorder>
   <zorder>three</zorder>
   <zorder>four</zorder>
   <zorder>five</zorder>
   <zorder>six</zorder>
   <zorder>seven</zorder>
   <zorder>eight</zorder>
   <zorder>special</zorder>
   <zorder>text_area</zorder>
   <zorder>plus</zorder>
   <zorder>minus</zorder>
   <zorder>mul</zorder>
   <zorder>div</zorder>
   <zorder>mod</zorder>
   <zorder>l_bracket</zorder>
   <zorder>r_bracket</zorder>
   <zorder>sqrt</zorder>
   <zorder>degree</zorder>
   <zorder>arrow</zorder>
   <zorder>x_sym</zorder>
   <zorder>ln</zorder>
   <zorder>log</zorder>
   <zorder>sin</zorder>
   <zorder>asin</zorder>
   <zorder>cos</zorder>
   <zorder>acos</zorder>
   <zorder>tan</zorder>
   <zorder>atan</zorder>
   <zorder>graph</zorder>
   <zorder>dot</zorder>
   <zorder>x_value</zorder>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>24</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_design">
    <property name="title">
     <string>Design</string>
    </property>
    <addaction name="SeaMelody"/>
    <addaction name="Koi"/>
    <addaction name="Panda"/>
    <addaction name="Sakura"/>
   </widget>
   <widget class="QMenu" name="menu_calc">
    <property name="title">
     <string>Calculator</string>
    </property>
    <addaction name="Math"/>
    <addaction name="Credit"/>
    <addaction name="Deposit"/>
   </widget>
   <addaction name="menu_design"/>
   <addaction name="menu_calc"/>
  </widget>
  <action name="SeaMelody">
   <property name="text">
    <string>Мелодия Моря</string>
   </property>
  </action>
  <action name="Koi">
   <property name="text">
    <string>Рыбка Кои</string>
   </property>
  </action>
  <action name="Panda">
   <property name="text">
    <string>Панда</string>
   </property>
  </action>
  <action name="Sakura">
   <property name="text">
    <string>Сакура</string>
   </property>
  </action>
  <action name="Credit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="Deposit">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
  <action name="Math">
   <property name="text">
    <string>Calculator</string>
   </property>
  </action>
  <action name="actionwhat_about_now">
   <property name="text">
    <string>what about now</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::ColorButton</class>
   <extends>QPushButton</extends>
   <header>Other/ColorButton.h</header>
  </customwidget>
  <customwidget>
   <class>s21::Button</class>
   <extends>QPushButton</extends>
   <header>Other/Button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
